### ISSUE TITLE
'property' deprecated in favour of 'choice_label'

### ISSUE THREAD

**Issue by hisie** [kbond/ZenstruckFormBundle#42]
In the symfony 2.7 documentation: https://github.com/symfony/symfony/blob/2.7/UPGRADE-2.7.md#form there is a change in the name of the label for the select options.

Are you going to change it to follow the same standard?
Thank you.

**Comment by kbond** [comment]
Yeah, it's probably a good idea to have both possible options (try to use `choice_label` and fallback to `property`).

Could you make a PR for this?

**Comment by hisie** [comment]
I hope i can do something this week.
